creativecloud/blocks/interactive-marquee/interactive-marquee.js

```diff
@@ -98,6 +98,20 @@ export default async function init(el) {
       await decorateGenfill(el, { createTag });
       break;
     }
+    case el.classList.contains('ff-masonry'): {
+      try {
+        const stylePromise = new Promise((resolve) => {
+          loadStyle('/creativecloud/features/firefly/firefly-masonry.css', resolve);
+        });
+        await Promise.all([stylePromise]);
+        interactiveInit(el, decorateButtons, decorateBlockBg, createTag);
+        const { default: setMultiImageMarquee } = await import('../../features/firefly/firefly-masonry.js');
+        await setMultiImageMarquee(el, { createIntersectionObserver, createTag });
```

Comment:
We can import createIntersectionObserver, createTag in the file from utils and do not need to pass them